test-apps/ui-test-app/src/frontend/appui/LocalFileSupport.ts

```diff
@@ -26,22 +26,32 @@ export class LocalFileSupport {
     let iModelConnection: IModelConnection | undefined;
     const filePath = `${SampleAppIModelApp.testAppConfiguration?.snapshotPath}/${fileName}`;
 
-    // open the imodel
-    Logger.logInfo(SampleAppIModelApp.loggerCategory(LocalFileSupport), `openLocalFile: Opening standalone or snapshot. path=${filePath} writable=${writable}`);
-
-    try {
-      iModelConnection = await BriefcaseConnection.openStandalone(filePath, writable ? OpenMode.ReadWrite : OpenMode.Readonly, { key: filePath });
-    } catch (err) {
-      Logger.logError(SampleAppIModelApp.loggerCategory(LocalFileSupport), `openLocalFile: BriefcaseConnection.openStandalone failed.`);
-
-      if (writable && err instanceof IModelError && err.errorNumber === IModelStatus.ReadOnly) {
+    // Open the iModel
+    if (ProcessDetector.isElectronAppFrontend) {
+      Logger.logInfo(SampleAppIModelApp.loggerCategory(LocalFileSupport), `openLocalFile: Opening standalone. path=${filePath} writable=${writable}`);
+      try {
+        iModelConnection = await BriefcaseConnection.openStandalone(filePath, writable ? OpenMode.ReadWrite : OpenMode.Readonly, { key: filePath });
+      } catch (err) {
+        Logger.logError(SampleAppIModelApp.loggerCategory(LocalFileSupport), `openLocalFile: BriefcaseConnection.openStandalone failed.`);
+
+        if (writable && err instanceof IModelError && err.errorNumber === IModelStatus.ReadOnly) {
+          iModelConnection = await SnapshotConnection.openFile(filePath);

```

Comment:
@bsteinbk If you want to allow interactive editing of this file, you should use `BriefcaseConnection.openStandalone`

Reply:
Before my change this morning to only call BriefcaseConnection.openStandalone for Electron, we were getting this error message in the catch:
Cannot read property 'invoke' of undefined